delta-lake/common/src/main/delta-33x-40x/scala/com/nvidia/spark/rapids/delta/common/GpuDeltaParquetFileFormatBase.scala

```diff
@@ -57,6 +57,11 @@ class GpuDeltaParquetFileFormatBase(
     isCDCRead: Boolean = false
   ) extends com.nvidia.spark.rapids.delta.GpuDeltaParquetFileFormat with Logging {
 
+  if (hasTablePath) {
+    logWarning("Input Delta table has deletion vectors. Optimizations such as file splitting " +
+      "and predicate pushdown are currently not supported for this table.")
```

Comment:
DVs are still off by default in OSS Delta. The warning does not read actionable. Since you insist on keeping it, it should tell the user why is it bad, maybe just point to the original delta issue / PR fixing it Otherwise it becomes super long. And suggest the mitigation is to disable DV and run optimize.